bm_scrapy/commands/describe_project.py

```diff
@@ -47,4 +46,4 @@ def run(self, args, opts):
                 universal_newlines=True,
             )
             result["image"] = output
-        logger(json.dumps(result))
+        print(json.dumps(result))
```

Comment:
@rodp63 why `print`?

Reply:
In the cli, we use docker to get the spider list, the returned value is the stdout of the container, no the logs.